src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
Can we remove the class, and use either functions from ``core/math/random.hpp``, ``arma::randn``, etc.?